src/internal/components/drawer/styles.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+/* stylelint-disable @cloudscape-design/no-implicit-descendant, selector-max-type */
+@use '../../styles/tokens' as awsui;
+@use '../../styles' as styles;
+
+.drawer {
+  @include styles.styles-reset;
+  word-wrap: break-word;
+  padding: awsui.$space-scaled-l awsui.$space-panel-side-right awsui.$space-scaled-xxxl awsui.$space-panel-side-left;
```

Comment:
Why does the help panel have zero bottom padding and the drawer `$space-scaled-xxxl`?

Reply:
Help panel has built-in padding for the last item (see [here](https://github.com/cloudscape-design/components/blob/main/src/help-panel/styles.scss#L76)), but since Drawers won't be content-based, the padding goes on the container.